pkg/option/config.go

```diff
@@ -2999,6 +3020,7 @@ func (c *DaemonConfig) Populate() {
 	c.HTTP403Message = viper.GetString(HTTP403Message)
 	c.DisableEnvoyVersionCheck = viper.GetBool(DisableEnvoyVersionCheck)
 	c.K8sNamespace = viper.GetString(K8sNamespaceName)
+	c.AgentNotReadyNodeTaintPrefix = viper.GetString(AgentNotReadyNodeTaintPrefixName)
```

Comment:
as this flag is _hidden_, I reckon you need to use helm extraConfig param for setting it, right ?

Reply:
I think that's right from reading, but I'm not actually sure whether that's desirable behavior. I'm still working on a test of this on my own infra to verify, so I may make changes based on how that goes (was building images on top of the 1.11.2 tag to test with).